qubes-rpc/qubes.GetAppmenus

```diff
@@ -6,6 +6,16 @@
 #  - directories stored on /rw in case of "rw-only" persistence
 #  - nothing, otherwise
 
+# Reload scripts in /etc/profile.d/, in case they register additional
+# directories in XDG_DATA_DIRS and we forgot them
+# (e.g. because we are running under sudo).
+for i in /etc/profile.d/*.sh ; do
+    if [ -r "$i" ]; then
+	# shellcheck disable=SC1090
```

Comment:
Indentation is odd (3 instead of 4 spaces).

Reply:
Ouch, that's a tab, my editor misbehaved. I pushed an updated commit.